packages/atomic/src/components/search/facets/atomic-popover/atomic-popover.pcss

```diff
@@ -0,0 +1,6 @@
+@import '../../../../global/global.pcss';
+@import '../facet-common.pcss';
+
+.popover-wrapper .slot-wrapper.selected {
+  display: block;
+}
```

Comment:
Could you remove this & just put the `block` tailwind class instead of selected?